automatic/openssl.light/tools/chocolateyInstall.ps1

```diff
@@ -20,5 +25,7 @@ $packageArgs = @{
 }
 
 Install-ChocolateyInstallPackage @packageArgs
+$binPath = "$($dir)\bin" 
+Install-ChocolateyPath -PathToInstall $binPath 
```

Comment:
> Just spitballing, is this the right way to add to the path variable in a choco package?

I've not used the `Install-ChocolateyPath` that often myself, so I can't remember exactly.
But yes, I believe so.